### PR TITLE
Add database subcategory

### PR DESCRIPTION
**Pull request by thoughtkettle**
* Part I of recategorization
* redirects included

### CODE REVIEW

File: _classic/databases/cassandra.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - databases
   - cassandra
-category: Getting Started
+category: Databases
 redirect_from:
   - /databases/cassandra/
+  - /classic/getting-started/cassandra
```
Comment (mlocher) [inline _classic/databases/cassandra.md]: please include trailing slashes on the URLS (see https://github.com/jekyll/jekyll-redirect-from#usage)

File: _classic/databases/mongodb.md [hunk 2]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - databases
   - mongodb
-category: Getting Started
+category: Databases
 redirect_from:
   - /databases/mongodb/
+  - /classic/getting-started/mongodb
```
Comment (mlocher) [inline _classic/databases/mongodb.md]: trailing slash :)

File: _classic/databases/mysql.md [hunk 3]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - databases
   - mysql
-category: Getting Started
+category: Databases
 redirect_from:
   - /databases/mysql/
+  - /classic/getting-started/mysql
```
Comment (mlocher) [inline _classic/databases/mysql.md]: trailing slash :)

File: _classic/databases/postgresql.md [hunk 4]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - databases
   - postgresql
-category: Getting Started
+category: Databases
 redirect_from:
   - /databases/postgresql/
+  - /classic/getting-started/postgresql
```
Comment (mlocher) [inline _classic/databases/postgresql.md]: trailing slash :)

File: _classic/databases/rethinkdb.md [hunk 5]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - databases
   - rethinkdb
-category: Getting Started
+category: Databases
 redirect_from:
   - /databases/rethinkdb/
+  - /classic/getting-started/rethinkdb
```
Comment (mlocher) [inline _classic/databases/rethinkdb.md]: trailing slash :)

File: _classic/databases/sqlite.md [hunk 6]
```diff
@@ -5,6 +5,6 @@ tags:
   - services
   - databases
   - sqlite
-category: Getting Started
+category: Databases
```
Comment (mlocher) [inline _classic/databases/sqlite.md]: trailing slash :)

File: _classic/services/beanstalkd.md [hunk 7]
```diff
@@ -5,6 +5,6 @@ tags:
   - services
   - queues
   - beanstalkd
-category: Getting Started
+category: Services
```
Comment (mlocher) [inline _classic/services/beanstalkd.md]: We should add a redirect for this page as well.

Comment (mlocher) [inline _classic/services/beanstalkd.md]: And this should go to the _Databases_ category, see http://kr.github.io/beanstalkd/ (or a _Queues_ category, if we want to pull out _Redis_ and _RabbitMQ_ as well).

File: _classic/services/coveralls.md [hunk 8]
```diff
@@ -4,9 +4,10 @@ layout: page
 tags:
   - analytics
   - integrations
-category: Getting Started
+category: Services
 redirect_from:
   - /analytics/coveralls/
+  - /classic/getting-started/coveralls
```
Comment (mlocher) [inline _classic/services/coveralls.md]: trailing slash

File: _classic/services/elasticsearch.md [hunk 9]
```diff
@@ -4,9 +4,10 @@ layout: page
 tags:
   - services
   - elasticsearch
-category: Getting Started
+category: Services
 redirect_from:
   - /services/elasticsearch/
+  - /classic/getting-started/elasticsearch
```
Comment (mlocher) [inline _classic/services/elasticsearch.md]: trailing slash

File: _classic/services/elasticsearch.md [hunk 10]
```diff
@@ -4,9 +4,10 @@ layout: page
 tags:
   - services
   - elasticsearch
-category: Getting Started
+category: Services
```
Comment (mlocher) [inline _classic/services/elasticsearch.md]: Same as with Beanstalkd above, I wouldn't move this into _Services_

File: _classic/services/memcached.md [hunk 11]
```diff
@@ -4,8 +4,9 @@ layout: page
 tags:
   - services
   - memcached
-category: Getting Started
+category: Services
```
Comment (mlocher) [inline _classic/services/memcached.md]: Wouldn't move this into _Services_

File: _classic/services/memcached.md [hunk 12]
```diff
@@ -4,8 +4,9 @@ layout: page
...
 redirect_from:
   - /services/memcached/
+  - /classic/getting-started/memcached
```
Comment (mlocher) [inline _classic/services/memcached.md]: trailing slash

File: _classic/services/rabbitmq.md [hunk 13]
```diff
@@ -5,9 +5,10 @@ tags:
   - services
   - queues
   - rabbitmq
-category: Getting Started
+category: Services
```
Comment (mlocher) [inline _classic/services/rabbitmq.md]: Wouldn't move this into _Services_

File: _classic/services/rabbitmq.md [hunk 14]
```diff
@@ -5,9 +5,10 @@ tags:
...
 redirect_from:
   - /queues/rabbitmq/
+  - /classic/getting-started/rabbitmq
```
Comment (mlocher) [inline _classic/services/rabbitmq.md]: trailing slash

File: _classic/services/redis.md [hunk 15]
```diff
@@ -5,8 +5,9 @@ tags:
   - services
   - queues
   - redis
-category: Getting Started
+category: Services
```
Comment (mlocher) [inline _classic/services/redis.md]: Wouldn't move this into _Services_

File: _classic/services/redis.md [hunk 16]
```diff
@@ -5,8 +5,9 @@ tags:
...
 redirect_from:
   - /queues/redis/
+  - /classic/getting-started/redis
```
Comment (mlocher) [inline _classic/services/redis.md]: trailing slash

**Comment by thoughtkettle** [comment]
Closing this. Creating a new one.

**Pull request closed without merging**